src/components/ArticleCard.jsx

```diff
@@ -11,12 +11,15 @@ export default function ArticleCard({ articles }) {
               <img className="image" src={article.article_img_url} />
               <p>Author: {article.author}</p>
               <p>Title: {article.title}</p>
-              <p>Article id: {article.article_id}</p>
               <p>Topic: {article.topic}</p>
               <p>Created at: {article.created_at}</p>
               <p>Votes: {article.votes}</p>
               <p>Comment count: {article.comment_count}</p>
-              <button>View</button>
+              <p>
+                <Link to={`/articles/${article.article_id}`}>
+                  Link to Article: {i + 1}
+                </Link>
+              </p>
```

Comment:
Well done implementing the `<Link>` tag. On a side note - does the user need to know the article id? Or would 'Link to article' be fine as it is?